bsi/src/main/java/org/roaringbitmap/bsi/longlong/Roaring64NavigableMapSliceIndex.java

```diff
@@ -0,0 +1,760 @@
+package org.roaringbitmap.bsi.longlong;
+
+import static org.roaringbitmap.longlong.Roaring64NavigableMap.SERIALIZATION_MODE_LEGACY;
```

Comment:
I would recommand not using static imports.

Reply:
ok